image/structure/image_layout.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2016 The Linux Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package structure
+
+// ImageLayoutVersion represents the oci-version content for the image
+// layout format.
+type ImageLayoutVersion struct {
```

Comment:
On Mon, Jun 20, 2016 at 05:51:24AM -0700, Sergiusz Urbaniak wrote:
> > +// ImageLayoutVersion represents the oci-version content for the image
> > +// layout format.
> > +type ImageLayoutVersion struct {
> 
> Do we want these types really exported?

This is part of the spec [1], so I think we do want to export it.  We
probably also want a JSON Schema for that content as well, in which
case the public type will probably end up being generated by #147 or
similar.

[1]: https://github.com/opencontainers/image-spec/blame/c3ad3c723522b1155678b5c5c83971fc2a98d7ca/image-layout.md#L15